edge-functions/basic-auth-password/middleware.ts

```diff
@@ -1,21 +1,24 @@
+/* eslint-disable @next/next/no-server-import-in-page */
```

Comment:
Install the `canary` version of `eslint-config-next` and you should be able to remove this line.